services/handler.py

```diff
@@ -0,0 +1,30 @@
+import json
+from http.server import BaseHTTPRequestHandler
+
+
+class Handler(BaseHTTPRequestHandler):
+
+    def do_HEAD(self):
+        self.send_response(200)
+        self.send_header('Content-type', 'application/json')
+        self.end_headers()
+        
+
+    def do_POST(self):
+        if self.headers.get('Authorization') is None:
+            length = int(self.headers.get('Content-Length'))
+            data = self.rfile.read(length).decode('utf-8')
+
+            self.do_HEAD()
+            self.wfile.write(data.encode('utf-8'))
+            pass
+        else:
+            pass
+
+
+def get_content(path: str):
+    path = path.replace("+", " ")
+    result = dict((a.strip(), b.strip())
+                  for a, b in (element.split('=')
+                               for element in path.split('&')))
+    return result
```

Comment:
get_content not used, but something like this is more important for later implementation : 
```suggestion
def parse_request(request: dict):
    match true:
    	case 'widget' in request:
    		# return a `dict[string]Any`
    		handle_widget(request['widget'], request['data'], request['props'], data['context'])
    	case 'action' in request:
    		# no return value
    		handle_listener(request['widget'], request['props'], request['event'], request['api'])
    	case 'resource' in request:
    		# return a `[]byte`
    		handle_resource(request['resource'])
```

Reply:
I added this to use it in the next issue

Reply:
I intended to use that in later implementation, I will remove it for now and add it later